pkg/configprovider/opampprovider/testdata/config_new_merge_disabled.yaml

```diff
@@ -0,0 +1,5 @@
+extensions:
+  opamp:
+    endpoint: "wss://example.com/v1/opamp"
+    remote_configuration_directory: ../globprovider/testdata/mergefunc
+    new_configmergeflow_disabled: true
```

Comment:
I think this field could be named - `disable_tag_config_merge` or something similar to better represent it. Also need a documentation update for it along with the other params.